### PR TITLE
Ics siemens tia target

### PR DESCRIPTION
**Pull request by 00010111**
## Description

I have created a folder in Targets called "ICS" and created a target to collect the Settings.xml file written by the Siemens TIA Portal. I think a first ICS related target justifies the creation of a new folder. Settings.xml file contains information on which projects have been interacted with. In die documentation of the Target I added links to a series of blog posts and a tool to parse the Settings.xml file. Please note, the mentioned part 2 of the blog post series is about to be published in the upcoming two weeks. (I did not include the direct link to a series to avoid another pull request just to add a link) 

## Checklist:
Please replace every instance of `[ ]` with `[X]` OR click on the checkboxes after you submit your PR

- [X ] I have generated a unique `GUID` for my Target(s)/Module(s)
- [X] I have placed the Target(s)/Module(s) in an appropriate subfolder in Targets or Modules. If one doesn't exist, I have either added it to the `Misc` folder or created a relevant subfolder **with justification**
- [X] I have set or updated the version of my Target(s)/Module(s)
- [X] I have verified that KAPE parses the Target(s)/Module(s) successfully via kape.exe, using `--tlist`/`--mlist` and corrected any errors 
- [X] I have validated my Target(s)/Module(s) against test data and verified they are working as intended
- [X] I have made an attempt to document the artifacts within the Target(s) or Module(s) I am submitting. If documentation doesn't exist, I have placed `N/A` underneath the Documentation header
- [X] For Targets, I have consulted either the [Target Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/TargetGuide.guide), [Target Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/TargetTemplate.template), [Compound Target Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/CompoundTargetGuide.guide), or [Compound Target Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/CompoundTargetTemplate.template) to ensure my Target(s) follow the same format
- [ ] For Modules, I have consulted either the [Module Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/ModuleGuide.guide), [Module Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/ModuleTemplate.template), [Compound Module Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/CompoundModuleGuide.guide), or [Compound Module Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/CompoundModuleTemplate.template) to ensure my Module(s) follow the same format

If your submission involves an SQLite database, have you considered making an [SQLECmd Map](https://github.com/EricZimmerman/SQLECmd/tree/master/SQLMap/Maps) for the SQLite database? If you make a Map, please add the SQLite database to the [SQLiteDatabases.tkape Compound Target](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/Compound/SQLiteDatabases.tkape).

Thank you for your submission and for contributing to the DFIR community!

### CODE REVIEW

File: Targets/ICS/SiemensTIA.tkape [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    -
+        Name: Siemens TIA Settings
+        Category: ICS
+        Path: C:\Users\%user%\AppData\Roaming\Siemens\Automation\Portal*\Settings\
+        Recursive: true
+
+# Documentation
+# The Settings.xml file contains information about opened project files, last storge location used and so on.
+# You can find more info on the Settings.xml file in part 2 of this blog post series: https://blog.nviso.eu/series/investigating-an-engineering-workstation/  
+# There is a tool available to parse the Settings.xml file: https://github.com/00010111/parse_tSettings The verbose output also provides information on the content of the Settings.xml file
```
Comment (AndrewRathbun) [inline Targets/ICS/SiemensTIA.tkape:16]: Are we going to have a Module made for this tool?

Comment (00010111) [reply Targets/ICS/SiemensTIA.tkape]: If I find the time for yes. If someone else wants to give it a go, this is also fine with me.

**Comment by AndrewRathbun** [comment]
First of all, thanks for the contribution. 

Secondly, this will not be getting its own folder under `.\Targets`. It can live under Apps. Things only get their own folders in the Targets parent directory if they are a major, major category, of which ICS is not. Frankly, I don't even know what ICS stands for. But either way, for now it'll live under `.\Targets\Apps`. Within the Apps folder, if something has multiple (i.e., 3ish or more dedicated Targets, then it warrants having it's own subfolder under `.\Targets\Apps` as a general guideline. 

Thirdly, see my comment above and let me know your thoughts.

Lastly, I removed trailing whitespace which was tripping the YAML linter (think of lint on a suit you wear to a formal event, got to get rid of that. Same concept but for syntax).

**Comment by 00010111** [comment]
Thanks for the Review Andrew.
I'm fine with moving it under Apps. 
ICS -> Industrial Control Systems 
In regards of module: as stated, depends if I find the time, or someone else before me ;)

regarding the whitespace: I'll take a note on that for any future contribution.

**Comment by AndrewRathbun** [comment]
If you end up making more Targets for other ICS, then by all means, `.\Targets\Apps\ICS` can be a thing to store those similar/related Targets.

**Comment by 00010111** [comment]
would love to, but sometimes hard to get the specific software.
If so, i took note on App\ICS as a path ;)

**Review by AndrewRathbun — APPROVED** [review]

**Pull request merged**